src/Basis/Constructors.jl

```diff
@@ -342,16 +532,17 @@ Tensor product basis on Gauss-Legendre-Lobatto points with Gauss-Legendre (defau
 - `collocatedquadrature = false`:  Gauss-Legendre or Gauss-Legendre-Lobatto quadrature points,
                                        default: false, Gauss-Legendre-Lobatto
 
+# Keyword Arguments:
```

Comment:
This should be combined with the previous 'Keyword Arguments' block two lines above and align everything to the right of the `:` accordingly